presto-tests/src/test/java/com/facebook/presto/memory/TestMemoryManager.java

```diff
@@ -85,35 +88,46 @@ public void shutdown()
         executor = null;
     }
 
-    @Test(timeOut = 240_000)
-    public void testResourceOverCommit()
+    @AfterMethod
+    public void cleanup()
+    {
+        for (TestingPrestoServer server : queryRunner.getServers()) {
```

Comment:
This will be a problem if any group has more than one test.  And it doesn't close any other services associated with the query runner. How about instead having an AfterGroups method that closes the query runner when the group is done?